docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
+. Configure the core configuration for your connector:
+.. Provide a unique name for the connector. 
+.. Type the *Client ID* and *Client Secret* of the service account that you created for Connectors and then click *Next*.
 
-. In this step you provide the connector specific configuration.
-.. Leave the *Data shape Format* to `application/octet-stream`
-.. *Method*: Leave to `POST`.
+. Provide connector-specific configuration:
+.. *Data shape Format*: Accept the default, `application/octet-stream`.
+.. *Method*: Accept the default, `POST`.
 .. *URL*: Enter your unique URL from link:https://webhook.site[webhook.site^].
-.. *Topic Names*: Enter the name of the topic you created in the first task of this quick start. Use the same topic as for the data sink connector.
+.. *Topic Names*: Type the name of the topic that you used for the source Connectors instance (for the quick start example, type *test-topic*).
 
-. Finally configure the error handler for the connector. You can choose between *stop* (the connector shuts down in case of errors), *log* (the error is logged) or *dead letter queue* (the events that cannot be handled by the connector are sent to a dead letter topic of your Kafka instance).
-Select `log`.
+. Optionally, configure the error handling policy for your Connectors instance. For the quick start, select *log*  and then click *Next*.
 
-. The next screen shows the configuration properties of your connector. Click *Create connector* to proceed with the deployment of the connector.
+.. Review the summary of the configuration properties of your Connectors instance and then click *Create Connectors instance* to deploy it.
 +
-After you complete the connector setup, the new connector is listed in the connectors table. After a couple of seconds, the connector moves to the *Ready* state. At this point the connector starts consuming messages from the Kafka topic associated with the connector and sending them to the HTTP sink.
+Your Connectors instance is listed in the table of Connectors. After a couple of seconds, the status of your Connectors instance changes to the *Ready* state and it consuming messages from the associated Kafka topic and sends them to the data sink (for the quick start, the data sink is the HTTP url that you provided).
 
-. In the browser tab pointing to link:https://webhook.site[webhook.site^] you should see the HTTP POST calls from the connector with the message contents as defined in the source connector.
+. Open the browser tab to link:https://webhook.site[webhook.site^] to see the HTTP POST calls with the message contents defined in the source connector.
```

Comment:
```suggestion
. Open a new web browser tab and navigate to link:https://webhook.site[webhook.site^] to see the HTTP POST calls with the message contents defined in the source connector.
```

Reply:
it needs to be the same browser tab - ot I have to reword this - will add a comment to deal with it later